app/helpers/helpers.rb

```diff
@@ -0,0 +1,42 @@
+helpers do
+
+  def prepare_feed
+    @all_peeps = Peep.all
+    @all_peeps.reverse!
+    @all_users = User.all
+  end
+
+  def prepare_public_feed
+    prepare_feed
+    @public_feed = []
+    @all_peeps.each do |peep|
```

Comment:
Avoid loading unnecessary records from the database then filtering them in the model layer.  Can you use DataMapper instead to access only public peeps?